webcompat/templates/home-page/form.html

```diff
@@ -89,41 +89,38 @@ <h1 class="headline-1">Report Site Issue</h1>
       {% endfor %}
     </div>
     <div class="form-upload form-element js-image-upload js-Form-group">
-      <label class="wc-UploadForm wc-UploadForm--homePage js-image-upload-label" for="image">
```

Comment:
Interesting fact: we had `<label for=image>`, but the associated `<input>` never had an `id=image`. But it didn't matter because the input is associated by the label that wraps it anyways. 🌠